src/AppInstallerRepositoryCore/Rest/Schema/1_0/Interface.cpp

```diff
@@ -54,11 +54,12 @@ namespace AppInstaller::Repository::Rest::Schema::V1_0
         }
     }
 
-    Interface::Interface(const std::string& restApi)
+    Interface::Interface(const std::string& restApi, const HttpClientHelper& httpClientHelper)
     {
         THROW_HR_IF(APPINSTALLER_CLI_ERROR_RESTSOURCE_INVALID_URL, !RestHelper::IsValidUri(JsonHelper::GetUtilityString(restApi)));
 
         m_restApiUri = restApi;
+        m_httpClientHelper = httpClientHelper;

```

Comment:
nit: could use initializer

Reply:
I meant m_restApiUri too, either use or not use initializer for both

---
In reply to: [612847093](https://github.com/microsoft/winget-cli/pull/861#discussion_r612847093) [](ancestors = 612847093)